extensions/standard-processors/processors/InvokeHTTP.h

```diff
@@ -187,7 +177,7 @@ class InvokeHTTP : public core::Processor {
       .withPropertyType(core::StandardPropertyTypes::DATA_TRANSFER_SPEED_TYPE)
       .build();
 
-  EXTENSIONAPI static constexpr auto Properties = std::array<core::PropertyReference, 23>{
+  EXTENSIONAPI static constexpr auto Properties = std::array<core::PropertyReference, 21>{
```

Comment:
Are there alternatives we can have instead of hardcoded property reference count for the array? I see this being done in multiple processor header files.

Reply:
You are right that this is ugly; it could be changed to `std::to_array<core::PropertyReference>({Method, URL, ...})`. (Assuming all our compilers support `to_array`, but I'd be surprised if they didn't.)

If this is changed, it should be changed everywhere and in a separate pull request. If you create the PR, I'd be happy to approve it.

Reply:
@fgerlits Thank you for sharing your perspective on refactoring this part of the code. I'd be happy to create a separate PR targetting your Jira MINIFICPP-2441. I will get back to you on this.

Reply:
@fgerlits @martinzink I opened the PR that updates the way most MiNiFi C++ Processor and ControllerService Properties are created: https://github.com/apache/nifi-minifi-cpp/pull/1859